src/main/java/org/jetbrains/research/refactorinsight/folding/handlers/MoveOperationFoldingHandler.java

```diff
@@ -18,11 +19,11 @@
     if (method == null) {
       return Collections.emptyList();
     }
-    String details = isBefore ? info.getDetailsAfter() : info.getDetailsBefore();
+    String details = isBefore ? info.getLeftPath() : info.getRightPath();
     String hintText = specificOperation(info.getType())
-        + (info.isChanged() ? " with changes " : " without changes ")
-        + (isBefore ? "to " : "from ")
-        + details.substring(details.lastIndexOf('.') + 1);
+        + (isBefore ? " to " : " from ")
+        + details.substring(details.lastIndexOf(File.separatorChar) + 1)
+        + (info.isChanged() ? ". With changes." : ". Without changes.");
```

Comment:
Yet again, the hint should look like "Moved to A.java **with changes**" or "Moved to A.java **with no changes**". There is no need to split it into several sentences.